package/com.unity.formats.usd/package.json

```diff
@@ -1,6 +1,8 @@
 {
     "dependencies": {
-        "com.unity.timeline" : "1.0.0"
+        "com.unity.timeline" : "1.0.0",
+        "com.unity.modules.screencapture": "1.0.0",
+        "com.unity.modules.physics": "1.0.0"
```

Comment:
for my curiosity, what part of physics is USD using ?

Reply:
RigidBody